codecov/codecov-python
# Add support for Python 3.7

Add support for Python 3.7:

* Test on Travis CI
  * Xenial is required for 3.7 on Travis
  * Add `python_requires` to help pip install the correct version for the running Python
  * Update Trove classifiers

Also:

* Remove redundant code for EOL Python 2.6
* Some tidy up of Travis CI config:
  * sudo no longer required
    * https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration
  * Drop the dot in py.test
    * https://twitter.com/pytestdotorg/status/753767547866972160
  * Cache downloaded pip files

AppVeyor config not touched, that will be fixed by PR https://github.com/codecov/codecov-python/pull/190.
